app/packages/looker-3d/src/fo3d/point-cloud/Pcd.tsx

```diff
@@ -32,7 +32,19 @@ export const Pcd = ({
     [pcdPath, preTransformedPcdPath, fo3dRoot]
   );
 
-  const points = useLoader(PCDLoader, pcdUrl);
+  const points_ = useLoader(PCDLoader, pcdUrl);
+
+  // todo: hack until https://github.com/pmndrs/react-three-fiber/issues/245 is fixed
+  const points = useMemo(() => points_.clone(false), [points_]);
+
+  useEffect(() => {
+    if (!points) {
+      return;
+    }
+
+    points.geometry.center();
```

Comment:
```suggestion
 ```suggestion
      if (points) {
        points.geometry.center();
      }
```
?